content/en/internal/configuration-database/_index.md

```diff
@@ -1,44 +1,27 @@
 ---
 title: "Configuration Database"
-description: "Configuration Database (GOCDB) documentation"
-weight: 30
+description: "Service configuration database"
+weight: 20
 type: "docs"
 ---
 
-{{% pageinfo %}} This documentation about the Configuration Database is a work
-in progress, more documentation is currently available at
-[EGI's wiki](https://wiki.egi.eu/wiki/GOCDB). {{% /pageinfo %}}
+## What is it?
 
-The EGI Configuration Database (previously known as the GOCDB) contains general
-information about the sites participating to the production infrastructure.
+The [EGI Configuration Database](https://goc.egi.eu) (GOCDB) is a
+**central registry that records topology information about all sites**
+participating in the EGI Cloud infrastructure.
 
-It's a central registry to record information about the topology of the EGI
-e-Infrastructure. This includes entities such as Operations Centres, Resource
-Centres, service endpoints and their downtimes, contact information and roles of
-users responsible for operations at different levels.
+The configuration database also provides different rules and grouping
```

Comment:
I think that in this context Configuration Database should always be written with a capital letter.

Reply:
Okay, I will update this page. But let's have some sort of discussion about this. I find that people blindly overuse this "rule" and the result is excessive capitalization of nouns (one recent example is Pull Request).